pmm-app/tests/pages/qanPage.js

```diff
@@ -457,4 +470,78 @@ module.exports = {
     const pages = await I.grabTextFrom(pagesCount);
     return pages[pages.length - 1];
   },
+
+  addSpecificColumn(columnName) {
+    const column = `//li[contains(text(), '${columnName}')]`;
+    I.waitForVisible(column, 30);
+    I.click(column);
+  },
+
+  verifyAddedColumn(columnName) {
+    const columnHeader = `//span[contains(text(), '${columnName}')]`;
+    I.waitForVisible(columnHeader, 30);
+    I.seeElement(columnHeader);
+  },
+
+  changeMetricInTable(columnName) {
+    const columnHeader = `//span[contains(text(), '${columnName}')]`;
+    I.waitForVisible(columnHeader, 30);
+    I.click(columnHeader);
+  },
+
+  verifyChangedColumn(columnName) {
+    const columnHeader = `//span[contains(text(), '${columnName}')]`;
+    I.waitForInvisible(columnHeader, 30);
+  },
+
+  verifyAppliedSortingForColumn(column) {
+    const loadColumn = `//span[contains(text(), '${column}')]/ancestor::*[@class='ant-table-column-sorters']/descendant::i[contains(@class, 'anticon anticon-caret-down ant-table-column-sorter-down on')]`;
+    I.waitForVisible(loadColumn, 30);
+  },
+
+  getColumn(column) {
+    return `//span[contains(text(), '${column}')]`;
+  },
+
+  getRow(row) {
+    return `//td[@class='ant-table-row-cell-break-word']//div[contains(text(), '${row}')]`;
+  },
+
+  getQueryCount(row) {
+    const time = '//table/tbody/tr[2]/td[5]/div/span/div/span';
+  },
+
+  async verifyQueryTime(queryTime) {
+    const queryTimeDetial = await I.grabTextFrom(this.fields.queryTimeDetail);
+    assert.equal(queryTimeDetial, queryTime, 'Query times are not same!');
+  },
+
+  async verifyAvqQueryCount() {
+    const queryTimeDetial = await I.grabTextFrom(this.fields.queryTimeDetail);
+    const queryCountDetail = await I.grabTextFrom(this.fields.queryCountDetail);
+    console.log('count ' + parseFloat(queryTimeDetial));
+    const result = parseFloat(queryCountDetail) / 300;
+    const roundedResult = Math.round(parseFloat(result) * 10) / 10;
+    console.log('result: ' + result);
+    console.log('count ' + parseInt(roundedResult));
+  },
+
+  async verifyAvgQueryTime() {
+    const queryTimeDetail = await I.grabTextFrom(this.fields.queryTimeDetail);
+    const queryCountDetail = await I.grabTextFrom(this.fields.queryCountDetail);
+    const load = await I.grabTextFrom(this.fields.load);
+    const result = (parseFloat(queryCountDetail) * (parseFloat(queryTimeDetail) / 1000)) / 300;
+    const roundedResult = Math.round(parseFloat(result) * 100) / 100;
+    assert.equal(roundedResult, parseFloat(load), 'Load should be same!');
+  },
+
+  async verifyAvgLockTime() {
+    const avgLoad = await I.grabTextFrom(this.fields.avgLoad);
+    const lockTimeDetail = await I.grabTextFrom(this.fields.lockTimeDetail);
+    const queryCountDetail = await I.grabTextFrom(this.fields.queryCountDetail);
+    const result = (parseFloat(queryCountDetail) * (parseFloat(lockTimeDetail) / 1000000)) / 300;
```

Comment:
why /1000000 here? this could be true in your specific example and instance, but we should really try to read the unit and then decide.